src/main/java/uk/nhs/hee/tis/trainee/sync/service/CurriculumMembershipSyncService.java

```diff
@@ -32,8 +31,10 @@
 import uk.nhs.hee.tis.trainee.sync.model.CurriculumMembership;
 import uk.nhs.hee.tis.trainee.sync.model.Record;
 import uk.nhs.hee.tis.trainee.sync.repository.CurriculumMembershipRepository;
+import static uk.nhs.hee.tis.trainee.sync.model.Operation.DELETE;
```

Comment:
Just had a look at the advice from SonarCloud CheckStyle, static import would be better to be placed on the top before other import
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=310&id=Health-Education-England_tis-trainee-sync&open=AYaYjwWahtDTow3ZJsGs&tab=why